docs/features/file-system.md

```diff
@@ -0,0 +1,181 @@
+# File System
+
+[[toc]]
+
+Mantle includes the [Flysystem](https://github.com/thephpleague/flysystem)
```

Comment:
Yep, definitely read Filesystem. and PHPlague instead of thephpleague.

Aanywho..